docusaurus.config.js

```diff
@@ -406,11 +406,11 @@ const config = {
           },
           3.16: {
             label: '3.16',
```

Comment:
3.15 did not have a label, does it matter that v3.16 does?

Reply:
Nope, doesn't get used unless multiple versions are being published. But I've removed it anyway. Thanks!